etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/RemoveObservationRequestsTest.groovy

```diff
@@ -40,8 +40,36 @@ class RemoveObservationRequestsTest extends Specification {
         thrown(ClassCastException)
     }
 
+    def "remove all LA Ochsner's OBRs except for the one with OBR-4.1 = '54089-8'"() {
+        given:
+        def fhirResource = ExamplesHelper.getExampleFhirResource("../LA/002_LA_ORU_R01_ORU_removal_test_file.fhir")
+        def bundle = fhirResource.getUnderlyingData() as Bundle
+
+        def initialDiagnosticReports = HapiHelper.resourcesInBundle(bundle, DiagnosticReport).toList()
+        def initialServiceRequests = HapiHelper.resourcesInBundle(bundle, ServiceRequest).toList()
+        def initialObservations = HapiHelper.resourcesInBundle(bundle, Observation).toList()
+        def obr4_1Values = initialServiceRequests.collect { HapiHelper.getOBR4_1Value(it) }
+
+        expect:
+        initialDiagnosticReports.size() > 1
+        initialServiceRequests.size() > 1
+        initialObservations.size() > 1
+        obr4_1 in obr4_1Values
+
+        when:
+        transformClass.transform(fhirResource, args)
+        def diagnosticReports = HapiHelper.resourcesInBundle(bundle, DiagnosticReport).toList()
+        def serviceRequests = HapiHelper.resourcesInBundle(bundle, ServiceRequest).toList()
+        def dr = diagnosticReports.first() as DiagnosticReport
+        def sr = HapiHelper.getServiceRequest(dr)
+
+        then:
+        diagnosticReports.size() == 1
+        serviceRequests.size() == 1
+        HapiHelper.getOBR4_1Value(sr) == obr4_1
+    }
 
-    def "remove all OBRs except for the one with OBR-4.1 = '54089-8'"() {
+    def "remove all CA USCD's OBRs except for the one with OBR-4.1 = '54089-8'"() {
```

Comment:
I believe this is a typo, UCSD is the name of the hospital not USCD

Reply:
Nice catch, I think it was more that I renamed it improperly since we have LA and CA OBR flattening now. I'll change it to just say `CA`

Reply:
Now that I look at the the [transformation_definitions](https://github.com/CDCgov/trusted-intermediary/blob/main/etor/src/main/resources/transformation_definitions.json#L157) again, the flattening is called `ucsdOruRemoveObservationRequests` and I believe the filtering is hospital specific, so the naming is intentional

Reply:
Right but you have uSCd vs uCSd, I think the middle two letters are transposed

Reply:
Ah, I missed that oops